q2_sample_classifier/__init__.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # ----------------------------------------------------------------------------
 # Copyright (c) 2017--, q2-sample-classifier development team.
```

Comment:
:+1:, I think we should standardize on the copyright notice we're using in other repos (at least until there's a need to be more general).